src/Compilers/VisualBasic/BasicAnalyzerDriver/VisualBasicDeclarationComputer.vb

```diff
@@ -61,36 +61,52 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     For Each decl In t.Members
                         ComputeDeclarationsCore(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken)
                     Next
-                    builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken))
+                    Dim attributes = GetAttributes(t.EnumStatement.AttributeLists)
+                    builder.Add(GetDeclarationInfo(model, node, getSymbol, attributes, cancellationToken))
+                    Return
+                Case SyntaxKind.EnumStatement
+                    Dim t = CType(node, EnumStatementSyntax)
```

Comment:
>CType [](start = 28, length = 5)

In cases like this, we usually use DirectCast.

Reply:
Perhaps should follow up later.

---
In reply to: [152415180](https://github.com/dotnet/roslyn/pull/23318#discussion_r152415180) [](ancestors = 152415180)

Reply:
Tracked with [https://github.com/dotnet/roslyn/issues/23334#issuecomment-346197831](https://github.com/dotnet/roslyn/issues/23334#issuecomment-346197831)

---
In reply to: [152415323](https://github.com/dotnet/roslyn/pull/23318#discussion_r152415323) [](ancestors = 152415323,152415180)